### PR TITLE
Update Hydro contract address on ethereum

### PR DESCRIPTION
**Pull request by Timidan**

### CODE REVIEW

File: blockchains/ethereum/assets/0x946112efaB61C3636CBD52DE2E1392D7A75A6f01/info.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Hydro Token",
```
Comment (vikmeup) [inline blockchains/ethereum/assets/0x946112efaB61C3636CBD52DE2E1392D7A75A6f01/info.json:2]: where to find official links to the contracts?

Comment (Timidan) [reply blockchains/ethereum/assets/0x946112efaB61C3636CBD52DE2E1392D7A75A6f01/info.json:2]: [Here](https://projecthydro.medium.com/hydro-smart-contract-redeployment-57bf0e1f3fb4) is an official statement on the redeployment process

[Here](https://etherscan.io/address/0x946112efab61c3636cbd52de2e1392d7a75a6f01#code) is the official newly deployed contract

Do tell me if you need any more info

Comment (Timidan) [reply blockchains/ethereum/assets/0x946112efaB61C3636CBD52DE2E1392D7A75A6f01/info.json:2]: Hi @vikmeup , do you need any more info about the new Hydro Token on eth?

**Comment by Timidan** [comment]
Hi guys, been a long time here, do you require me to do anything else?

**Comment by vikmeup** [comment]
As per our updated policy, brand new tokens are not included, there is a minimum requirement for token holder and number of transactions

See our [How to Get Submit Token Logo ](https://community.trustwallet.com/t/how-to-submit-a-token-logo/3863#how-to-get-approved) guidelines.

Feel free to re-open once you meet those requirements.

**Comment by Timidan** [comment]
Oh... I get this.... Will pass this on to the team ASAP

**Comment by catenocrypt** [comment]
New ERC20 contract added with #5957 . Closing

**Pull request closed without merging**